### ISSUE TITLE
Support "nicknames"

### ISSUE THREAD

**Issue by nattofriends** [bitlbee/bitlbee-facebook#93]
Messenger supports setting "nicknames" per chat. Can those be used for nicknames on Bitlbee side?

**Comment by dgw** [comment]
Do you mean nicknaming the chats or nicknaming users? Chat channel names can be set when adding them with `fbjoin`.

**Comment by nattofriends** [comment]
Nicknaming users.

2016-06-30 13:10 GMT-07:00 dgw <notifications@github.com>:

**Comment by dequis** [comment]
`help rename` in &bitlbee?

**Comment by nattofriends** [comment]
Hmm, I guess that works, but it would be nice to honor the Messenger-set nickname. (But might be hard because that can be set per-chat)

**Comment by dequis** [comment]
Indeed. IRC nicks are network-global, not chat-local. It gets dirty quick. XMPP has a similar problem.

I have a half-finished patch in my git stash to extend the `translate_to_nicks` feature to support chat-specific nicks. That feature modifies the outgoing message to have the correct nick. You can try that right now, if you type `JamesGeboski: blah` it gets sent as `James Geboski: blah`, with a space in the middle, assuming that JamesGeboski is a valid nick in your contact list. What's missing is a way to make it look up the nick specific to a groupchat.

Speaking of which, you might want to disable that if you're renaming people. `channel #whatever set translate_to_nicks off`
